slack-app-backend/src/main/java/com/slack/api/app_backend/events/servlet/SlackSignatureVerifier.java

```diff
@@ -5,6 +5,7 @@
 
 import javax.servlet.http.HttpServletRequest;
 
+@Deprecated // Please consider Bolt framework with bolt-servlet adapter instead
 @Slf4j
 public class SlackSignatureVerifier {
```

Comment:
This is used only by the above `SlackEventsApiServlet` class in this SDK. As this class is public, external projects may rely on it. Event after this deprecation, we can keep the class as long as possible (for the reason I mentioned above)